src/scripts/contributors/contributors.ts

```diff
@@ -1,121 +1,28 @@
-import { join } from 'node:path';
-import axios, { type AxiosRequestConfig } from 'axios';
-import { writeToFile } from '../../core';
-import { green, red, yellow } from '../helpers/painter';
-import { createScreenshot } from '../helpers/screenshots';
-import type { Contributor } from '../models/scripts/contributors/contributor';
-
-/**
- * Parse link header
- * @param linkHeader Link header as string
- * @returns Object that contains the page numbers of `prev`, `next` and `last`.
- */
-const parseLinkHeader = (linkHeader: string) => {
-  const nextPagePattern = new RegExp(/\bpage=(\d+)>;\srel="next"/);
-  const lastPagePattern = new RegExp(/\bpage=(\d+)>;\srel="last"/);
-  const prevPagePattern = new RegExp(/\bpage=(\d+)>;\srel="prev"/);
-
-  const nextPage = nextPagePattern.exec(linkHeader) ?? '';
-  const lastPage = lastPagePattern.exec(linkHeader) ?? '';
-  const prevPage = prevPagePattern.exec(linkHeader) ?? '';
-
-  return { nextPage, lastPage, prevPage };
-};
-
-/**
- * Get all contributors from GitHub API.
- */
-const fetchContributors = (
-  page: string
-): Promise<{ contributorsOfPage: Contributor[]; nextPage: string }> => {
-  return new Promise((resolve, reject) => {
-    const config: AxiosRequestConfig = {
-      method: 'get',
-      url: `https://api.github.com/repos/pkief/vscode-material-icon-theme/contributors`,
-      params: { page },
-      headers: {
-        accept: 'application/json',
-        'User-Agent': 'Contributors script',
-      },
-    };
-
-    axios
-      .request(config)
-      .then((res) => {
-        const { nextPage, lastPage, prevPage } = parseLinkHeader(
-          res.headers?.link?.toString() ?? ''
-        );
-        console.log(
-          '> Material Icon Theme:',
-          yellow(
-            `[${page}/${
-              lastPage ? lastPage[1] : +prevPage[1] + 1
-            }] Loading contributors from GitHub...`
-          )
-        );
-
-        resolve({ contributorsOfPage: res.data, nextPage: nextPage?.[1] });
-      })
-      .catch((err) => {
-        reject(err);
-      });
+const imageUrl = 'https://contributors-table.vercel.app/image';
```

Comment:
As already mentioned, I don't want to rely on a vercel service (as an external dependency and based on a private account) for something which is part of our build process. It doesn't make much sense to exclude the generation of the contributors image to a service. It was running perfectly fine in the past and there's no need to refactor this. I appreciate your work and the technical achievements behind it, but from compliance and release process perspective I don't like it.